File: stage1/problems/problem18/starter.cpp
#include <iostream>
#include <vector>
#include <string>
#include <sstream>

void solve() {
    // --- Input reading ---
    std::string line1, line2;
    std::getline(std::cin, line1);
    std::getline(std::cin, line2);
    
    std::stringstream ss1(line1);
    std::vector<int> a;
    int n;
    while(ss1 >> n) {
        a.push_back(n);
    }
    
    std::stringstream ss2(line2);
    std::vector<int> b;
    while(ss2 >> n) {
        b.push_back(n);
    }

    // --- Solution ---
    // The following variables are available:
    // a: std::vector<int>
    // b: std::vector<int>

    // TODO: Implement the solution
}

int main() {
    std::ios_base::sync_with_stdio(false);
    std::cin.tie(NULL);
    solve();
    return 0;
}

File: stage1/problems/problem18/starter.java
import java.util.*;

public class starter {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        
        // --- Input reading ---
        String[] line1 = sc.nextLine().split(" ");
        int[] a = new int[line1.length];
        for(int i = 0; i < line1.length; ++i) {
            a[i] = Integer.parseInt(line1[i]);
        }
        
        String[] line2 = sc.nextLine().split(" ");
        int[] b = new int[line2.length];
        for(int i = 0; i < line2.length; ++i) {
            b[i] = Integer.parseInt(line2[i]);
        }

        // --- Solution ---
        // The following variables are available:
        // a: int[]
        // b: int[]

        // TODO: Implement the solution
        
        sc.close();
    }
}
